_posts/2023-08-30-julia-solorzano.md

```diff
@@ -43,10 +43,12 @@ Julia's Talk is titled "Django the Dinosaur: A Story About A Web Framework."
 
 If you are interested in learning more about Julia, you can find her on the web at:
 
-[juliaelman.com](https://juliaelman.com/)
+[juliasolorzano.com](juliasolorzano.com/)
```

Comment:
This link seems to be redirecting to the older link, so might be worth keeping the old one?

Reply:
No, keep the new link because that's her new surname (recent marriage)